src/sql/workbench/browser/modal/modal.ts

```diff
@@ -491,7 +487,7 @@ export abstract class Modal extends Disposable implements IThemable {
 	/**
 	 * Return background color of header and footer
 	 */
-	protected get headerAndFooterBackground(): string {
+	protected get headerAndFooterBackground(): string | null {
```

Comment:
>null [](start = 53, length = 4)

Why null here vs undefined?

Reply:
This is assigned to an html property which are defined as null vs undefined.

Reply:
with strict nulls if the try and do `element.style.color = undefined` you will get an error since they are defined by typescript to be `string | null`